src/otx/core/model/entity/visual_prompting.py

```diff
@@ -57,13 +62,37 @@ def _export_parameters(self) -> dict[str, Any]:
         export_params = super()._export_parameters
         export_params["metadata"].update(
             {
-                ("model_info", "model_type"): "segment_anything",
+                ("model_info", "model_type"): "Visual_Prompting",
                 ("model_info", "task_type"): "visual_prompting",
             },
         )
         export_params["input_size"] = (1, 3, self.model.image_size, self.model.image_size)
+        export_params["resize_mode"] = "fit_to_window"
```

Comment:
Doesn't 'Pad` exist after resize? If it exists, need to change it to 'fit_to_window_letterbox'

Reply:
`fit_to_window` already do `pad` after `resize`.
https://github.com/openvinotoolkit/model_api/blob/41c20267122bed9fe9b21631166d15818f669379/model_api/python/model_api/adapters/utils.py#L358-L367

Reply:
As far as I know, "fit_to_window" executes https://github.com/openvinotoolkit/model_api/blob/41c20267122bed9fe9b21631166d15818f669379/model_api/python/model_api/adapters/utils.py#L417

Reply:
It is numpy version, but through `embed_preprocessing` in adapter, opset version will be used.

Reply:
But they do differently as I left a comment the below.